src/jitlayers.cpp

```diff
@@ -248,9 +248,12 @@ static jl_callptr_t _jl_compile_codeinst(
         IndirectCodeinsts += emitted.size() - 1;
     }
 
+    size_t i = 0;
     for (auto &def : emitted) {
         jl_code_instance_t *this_code = def.first;
-        jl_timing_show_func_sig(this_code->def->specTypes, JL_TIMING_CURRENT_BLOCK);
+        if (i < 10)
```

Comment:
Maybe make this controllable with an optional environment variable?

Reply:
Any recommendations about what to call it?

How about something fairly coarse like `JULIA_VERBOSE_TIMING_METADATA=1` that would turn off limits like this in general?

Reply:
JULIA_TIMING_METADATA_PRINT_LIMIT, which defaults to 10?